packages/grafana-ui/src/components/RefreshPicker/RefreshPicker.tsx

```diff
@@ -101,6 +101,13 @@ export class RefreshPicker extends PureComponent<Props> {
     );
     const ariaLabel = selectedValue.value === '' ? ariaLabelChooseIntervalMessage : ariaLabelDurationSelectedMessage;
 
+    const tooltipIntervalSelected = t(
+      'refresh-picker.tooltip.interval-selected',
+      `Auto refresh after ${durationAriaLabel}`
+    );
```

Comment:
see https://github.com/grafana/grafana/blob/laura/tooltip-for-auto-refresh/public/app/core/components/NestedFolderPicker/NestedFolderPicker.tsx#L230 for how to pass dynamic strings to translations